src/Homework3/Book.java

```diff
@@ -0,0 +1,54 @@
+package Homework3;
+
+import java.util.*;
+public class Book {
+    String ISBN;
+    String Name;
+    List<Author> Authors;
+    String Publisher;
+    String PublicationDate;
+    boolean Paperback;
+
+    Book() {
```

Comment:
Why this constructor is needed?

Reply:
It was added "just in case", deleted it.